Client/src/reducers/cityReducer.js

```diff
@@ -0,0 +1,22 @@
+
+
+const initState = {  //aqui creamos el estado que queremos cambiar con nuestra infomaracion nueva recogida de la base de datos, ya no crearemos state dentro de componentes.
+    cities: []
+}
+
+
+
+const cityReducer = (state = initState, action) => { //aqui creamos nuestro reducer, donde le vamos a crear el nuevo estado con la informacion de action que es donde diremos que lo que cambiamos o añadimos
+   console.log(action.payload);
+      if (action.type === "ALL_CITIES"){  //aqui hacemos un if donde diremos que si nuestro accion type es igual al que hemos puesto en nuestro accion nos devuelva toda la informacion
+            return{
+                cities: action.payload
```

Comment:
I think if you made this:
`cities: action.payload.cities`
you could avoid other places in the code where you are writing:
`this.props.myCities.cities`
Note that would be a breaking change and you would have to update the variable references like in the cities component.